mysql-test/lib/mtr_cases.pm

```diff
@@ -639,7 +639,7 @@ sub optimize_cases {
     # Check that engine selected by
     # --default-storage-engine=<engine> is supported
     # =======================================================
-    my %builtin_engines = ('myisam' => 1, 'memory' => 1, 'csv' => 1);
+    my %builtin_engines = ('myisam' => 1, 'memory' => 1, 'csv' => 1, 'rocksdb' => 1);
```

Comment:
So the proper fix would be to have all plugins loaded at mysql-test-run.pl:collect_mysqld_features. I have tried to implement exactly this for TokuDB with a fix for http://bugs.mysql.com/bug.php?id=72615 (https://bugs.launchpad.net/percona-server/+bug/1318537, and then https://bugs.launchpad.net/percona-server/+bug/1387227).

What does currently happen with RocksDB inside of collect_mysqld_features?